src/component/tooltip/TooltipHTMLContent.ts

```diff
@@ -461,15 +461,15 @@ class TooltipHTMLContent {
 
     getSize() {
         const el = this.el;
-        return [el.offsetWidth, el.offsetHeight];
+        return [el?.offsetWidth, el?.offsetHeight];
```

Comment:
We try to avoid `?.` in ECharts. Write explictly the expected value when `el` is not defined. Should it be something like `el ? [el.offsetWidth, el.offsetHeight] : [0, 0]` or what?

Reply:
I updated.

Reply:
Another reason is that ECharts will be eventually transpiled to ES3, and the optional chaining operator `?.` will be polyfilled to:
```js
return [el === null || el === void 0 ? void 0 : el.offsetWidth, el === null || el === void 0 ? void 0 : el.offsetHeight];
```
It causes an unexpected increment in code size, which ECharts tries to avoid until we target ECharts to ES2020.